casanovo/denovo/model_runner.py

```diff
@@ -105,15 +106,9 @@ def _execute_existing(
         out_filename=out_filename,
     )
     # Read the MS/MS spectra for which to predict peptide sequences.
-    if not os.path.isdir(peak_dir):
-        logger.error(
-            "Could not find directory %s from which to read peak files",
-            peak_dir,
-        )
-        raise FileNotFoundError(
-            "Could not find the directory to read peak files"
-        )
-    peak_filenames = _get_peak_filenames(peak_dir)
+    if len(peak_filenames := _get_peak_filenames(peak_path)) == 0:
```

Comment:
I'm not an extensive user of the walrus operator either. In long if-else statements it can be useful to make code more concise and help readability, although here it only saved us a single line.